suite-common/logger/src/logsSlice.ts

```diff
@@ -0,0 +1,42 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export type LogEntry = { datetime: string; type: any; payload?: Record<any, any> };
+
+type LogsSliceState = {
+    logs: LogEntry[];
+};
+
+type LogsSliceRootState = {
+    logs: LogsSliceState;
+};
+
+const logsSliceInitialState: LogsSliceState = {
+    logs: [],
+};
+
+const MAX_ENTRIES = 200;
+
+export const logsSlice = createSlice({
+    name: 'logs',
+    initialState: logsSliceInitialState,
+    reducers: {
+        addLog: (
+            state,
+            action: PayloadAction<{ action: any; payload: Record<string, unknown> | undefined }>,
+        ) => {
+            state.logs.push({
+                datetime: new Date().toUTCString(),
+                payload: action.payload.action.payload,
+                type: action.payload.action.type,
+                ...action.payload.action,
```

Comment:
This is wrong, we don't want to log the whole payload of the action, that is why there is an extra payload property. The naming is very confusing, but at the same time I don't know how to name action passed in payload of another action...

```suggestion
                type: action.payload.action.type,
                ...action.payload.payload,
```

Reply:
https://github.com/trezor/trezor-suite/pull/7359/commits/3cc634793eec06a7461fbad15de709a91906b19b

Reply:
This is still not resolved, you have to use `action.payload.payload` and not `action.payload.action`... payload.payload is a limited choice of properties from the whole payload of the original action.

Changes suggested by me wasn't correct, it should be:
```javascript
type: action.payload.action.type,
payload: action.payload.payload,
```